### PR TITLE
KMS-14957 show button on hover; show button on start

### PR DESCRIPTION
**Pull request by omridevk**

### CODE REVIEW

**Comment by omridevk** [comment]
no icon yet.

File: modules/CustomButton/resources/customButton.js [hunk 1]
```diff
@@ -3,6 +3,7 @@
 			//indicates we were explicitly asked to show the button (will be used when re-enabling the button)
 			shouldShow : false,
 			isDisabled: false,
+			ariaText: '',
```
Comment (atarsh) [inline modules/CustomButton/resources/customButton.js]: I'd add a default here from the plugin localization texts, maybe the same as the label?

**Comment by atarsh** [comment]
please ask Orem M to take a look, I'm not really in a position to code-review this :)

**Comment by omridevk** [comment]
@OrenMe 
can you please review this real quick?

Thanks a lot.

**Pull request merged**